src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/Details/DanishDetails.tsx

```diff
@@ -1,63 +1,77 @@
 import React from 'react'
 import { InputGroup } from 'components/inputs'
+import { useTextKeys } from 'utils/textKeys'
 import { isDanishHomeContentFieldSchema } from '../utils'
 import { DetailInput } from './components/DetailInput'
-import { SupportSection } from './components/SupportSection'
-import { Content, ContentColumn } from './components/Details.styles'
+import {
+  Content,
+  ContentColumn,
+  ContentColumnTitle,
+} from './components/Details.styles'
 import { DetailsProps } from './types'
+import { CommonFields } from './components/CommonFields'
 
 export const DanishDetails: React.FC<DetailsProps> = ({
   fieldSchema,
   formikProps,
   offerQuote,
-}) => (
```

Comment:
Not that much is changed here it just looks like it (same for the reset of the markets), basically: 
* Removed SupportSection
* Added CommonFields and Title for those.

(While developing this the design changed so now it is much easier to break out stuff, I can break out the 
Title and Common fields but I deemed it not necessary for this (atleast for this draft) so you can test it on your devices.)
Let me know your opinions